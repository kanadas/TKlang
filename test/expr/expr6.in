("alala", [2, 3, "lala"] ++ [] :: [int + string], ())
