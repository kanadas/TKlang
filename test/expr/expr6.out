expr/expr6.in
Error: Undefined case: ETuple (EString "alala") [EType (EConcat (EList [EInt 2,EInt 3,EString "lala"]) EEmpty) (TList (TUnion (TBasic (Basic "int")) (TIdent (Ident "string")))),EVoid]

[Abstract Syntax]

ETuple (EString "alala") [EType (EConcat (EList [EInt 2,EInt 3,EString "lala"]) EEmpty) (TList (TUnion (TBasic (Basic "int")) (TIdent (Ident "string")))),EVoid]

[Linearized tree]

("alala", [2, 3, "lala"] ++ [] :: [int + string], ())
